mods/rv/rules/ai.yaml

```diff
@@ -485,11 +490,11 @@ Player:
 			gaweap: 3
 			naweap: 3
 			yaweap: 3
-			# gadept: 1
-			# nadept: 1
-			# yadept: 1
-			gaairc: 1
-			# naheli: 1
+			gadept: 1
+			nadept: 1
+			yadept: 1
```

Comment:
I would keep the AI not built service depots until they can actually repair stuff. MCV in RV doesn't require SD and there is nothing else AI can use that requires SD.

Reply:
I will try let AI build expansion vehicle.

Reply:
add: for base push

Reply:
added AI control for expansion vehicle and added expansion vehicle to AI build list.

Going to add some defenses for AI that can be built when expansion vehicle deployed.